src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketCreate.java

```diff
@@ -67,13 +68,23 @@ public void run(ActionContext context) throws Exception {
 
     Configuration configuration = new Configuration();
 
-    String serviceAccountFilePath = config.getServiceAccountFilePath();
-    Credentials credentials = serviceAccountFilePath == null ?
-                                null : GCPUtils.loadServiceAccountCredentials(serviceAccountFilePath);
-
-    if (serviceAccountFilePath != null) {
-      configuration.set("google.cloud.auth.service.account.json.keyfile", serviceAccountFilePath);
+    Boolean isServiceAccountFilePath = config.isServiceAccountFilePath();
+    if (isServiceAccountFilePath == null) {
```

Comment:
check for null and empty

Reply:
@CuriousVini `isServiceAccountFilePath` is `Boolean` type.  Am I missing something ?